tests/components/Pagination.test.tsx

```diff
@@ -0,0 +1,120 @@
+import { render, screen, within } from '@testing-library/react';
+import { State, VerticalSearchState } from '@yext/answers-headless-react';
+import { spyOnActions, mockAnswersState, mockAnswersHooks } from '../__utils__/mocks';
+import { mockedVerticalResults } from '../__fixtures__/data/verticalresults';
+import userEvent from '@testing-library/user-event';
+import { Pagination } from '../../src/components/Pagination';
+
+const mockedState: Partial<State> = {
+  vertical: {
```

Comment:
I think some of the vertical state can be cut down now that we're only using the `Pagination` component. some of it (like `results`) was probably only needed for `VerticalResults`

Reply:
I'm taking a look at that at the moment-- I might be wrong, but I think the pagination component might behave differently depending on the presence of results or not, which requires looking at vertical state.  The first test is making me scratch my head a little because I think something might be going wrong from before, but I'm looking into it.